momentGW/util.py

```diff
@@ -456,7 +456,12 @@ def _fallback():
 
     # Reshape and transpose the output
     ct = ct.reshape(shape_ct, order="A")
-    c = ct.transpose(order_ct)
+    if ct.flags.f_contiguous:
+        c = np.asfortranarray(ct.transpose(order_ct))
+    elif ct.flags.c_contiguous:
+        c = np.ascontiguousarray(ct.transpose(order_ct))
+    else:
+        c = ct.transpose(order_ct)
```

Comment:
still not comfortable with this and don't think it's needed -- if there is need for contiguity it should be forced when needed and not here, is that ok?